tests/Terra/TWCoinTypeTests.cpp

```diff
@@ -13,7 +13,27 @@
 #include <gtest/gtest.h>
 
 
-TEST(TWTerraCoinType, TWCoinType) {
+TEST(TWTerraCoinType, TWCoinType20) {
+    auto symbol = WRAPS(TWCoinTypeConfigurationGetSymbol(TWCoinTypeTerraV2));
+    auto txId = WRAPS(TWStringCreateWithUTF8Bytes("D28D8AFC7CE89F2A22FA2DBF78D2C0A36E549BB830C4D9FA7459E3F723CA7182"));
+    auto txUrl = WRAPS(TWCoinTypeConfigurationGetTransactionURL(TWCoinTypeTerraV2, txId.get()));
+    auto accId = WRAPS(TWStringCreateWithUTF8Bytes("terra16t3gx5rqvz6ru37yzn3shuu20erv4ngmfr59zf"));
+    auto accUrl = WRAPS(TWCoinTypeConfigurationGetAccountURL(TWCoinTypeTerraV2, accId.get()));
+    auto id = WRAPS(TWCoinTypeConfigurationGetID(TWCoinTypeTerraV2));
+    auto name = WRAPS(TWCoinTypeConfigurationGetName(TWCoinTypeTerraV2));
+
+    ASSERT_EQ(TWCoinTypeConfigurationGetDecimals(TWCoinTypeTerraV2), 6);
+    ASSERT_EQ(TWBlockchainCosmos, TWCoinTypeBlockchain(TWCoinTypeTerraV2));
+    ASSERT_EQ(0x0, TWCoinTypeP2shPrefix(TWCoinTypeTerraV2));
+    ASSERT_EQ(0x0, TWCoinTypeStaticPrefix(TWCoinTypeTerraV2));
+    assertStringsEqual(symbol, "LUNA");
+    assertStringsEqual(txUrl, "https://finder.terra.money/tx/tx/D28D8AFC7CE89F2A22FA2DBF78D2C0A36E549BB830C4D9FA7459E3F723CA7182");
```

Comment:
should be `https://finder.terra.money/mainnet/tx/` instead of `https://finder.terra.money/tx/tx/`

Reply:
good catch, I will fix it